google/cloud/storage/examples/storage_object_samples.cc

```diff
@@ -135,12 +138,14 @@ void InsertObjectStrictIdempotency(google::cloud::storage::Client,
   (argv.at(0), argv.at(1), argv.at(2));
 }
 
+// NOLINTNEXTLINE(performance-unnecessary-value-param)
```

Comment:
The NOLINT is fine here, but I'm curious if you tried just declaring the first arg as a `const&`?